tests/test_other.py

```diff
@@ -7851,9 +7851,9 @@ def test(filename, expectations, size_slack):
 
       print('test on hello world')
       test(path_from_root('tests', 'hello_world.cpp'), [
-        ([],      21, ['assert'], ['waka'], 46505,  22,   16, 59), # noqa
-        (['-O1'], 16, ['assert'], ['waka'], 12630,  14,   14, 31), # noqa
-        (['-O2'], 16, ['assert'], ['waka'], 12616,  14,   14, 31), # noqa
+        ([],      22, ['assert'], ['waka'], 46505,  22,   16, 59), # noqa
+        (['-O1'], 17, ['assert'], ['waka'], 12630,  14,   14, 31), # noqa
+        (['-O2'], 17, ['assert'], ['waka'], 12616,  14,   14, 31), # noqa
```

Comment:
why has this increased?

Reply:
Adding `$allocateUTF8OnStack` to `DEFAULT_LIBRARY_FUNCS_TO_INCLUDE` has the effect that the function gets imported to asm.js/wasm module (even though it is not used there). I think meta-DCE in -O3 and -Os will then optimize it away.

This is a slight code size drawback for -O1 and -O2 iteration builds, but I don't believe anyone would use those builds to optimize code size, so that does not feel like a concern.

The benefit of moving these items away from preamble/shell/support/postamble runtime is that I can customize those files, and then explicitly set -s `DEFAULT_LIBRARY_FUNCS_TO_INCLUDE=[]` to ensure that nothing is pulled in automatically, but only via linking decisions as needed. Then only if I use a JS function that declares that it needs `$allocateUTF8OnStack`, it will be linked in.